.github/workflows/ci.yml

```diff
@@ -0,0 +1,44 @@
+name: Grid CI
+on: [push]
+jobs:
+  JSBuild:
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        node-version: [8.6.0]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Kahuna
+        working-directory: ./kahuna
+        run: |
+          npm install
+          npm test
+      - name: S3Watcher
+        working-directory: ./s3watcher/lambda
+        run: |
+          npm install
+          npm run build
+      - name: Reaper
+        working-directory: ./reaper
+        run: |
+          npm install
+          npm run riffraff-artefact
+  ScalaBuild:
+    runs-on: ubuntu-18.04
+    services:
+      elasticsearch:
+        image: elasticsearch:6.5.4
+        options: -e "discovery.type=single-node" --expose 9200 --health-cmd "curl localhost:9200/_cluster/health" --health-interval 10s --health-timeout 5s --health-retries 10
```

Comment:
Wait for elasticsearch to report healthy before continuing. Inspired by [this](https://github.com/actions/example-services/blob/master/.github/workflows/postgres-service.yml#L28) as it's not noted in the [docs](https://help.github.com/en/articles/workflow-syntax-for-github-actions#jobsjob_idservices).